lib/logger/lib/logger.ex

```diff
@@ -304,7 +304,8 @@ defmodule Logger do
   end
 
   @doc """
-  Adds the given keyword list to the current process metadata.
+  Adds the given keyword list to the current process metadata. This also allows you to 
```

Comment:
Well it does so in a nonstandard way. I could just break it into multiple paragraphs.